resources/application-connector/charts/connector-service/files/certs-restore.sh

```diff
@@ -0,0 +1,23 @@
+set -e
+set -o pipefail
+
+restore() {
+  namespace="$1"
+  secretName="$2"
+  backupSecretName="$secretName-backup"
+
+  echo "Restoring $backupSecretName"
+  secret=$(kubectl -n "$namespace" get secret "$backupSecretName" --ignore-not-found)
+  if [ -n "$secret" ]
+  then
+    echo "Removing $secretName"
+    kubectl delete secret "$secretName" -n="$namespace" --ignore-not-found
+    echo "$backupSecretName exists, reverting $secretName to $backupSecretName"
+    kubectl get secret "$backupSecretName" -n "$namespace" -oyaml | sed "s/name:.*$/name: $secretName/" | kubectl apply -n "$namespace" -f -
```

Comment:
You do not clean metadata yaml before import. This is not recommended way to export and import resources with the same metadata. Please use the method I have provided you in the example.